bucket/git-istage.json

```diff
@@ -0,0 +1,35 @@
+{
+    "homepage": "https://github.com/terrajobst/git-istage",
+    "version": "0.2.29",
+    "url": "https://www.nuget.org/api/v2/package/git-istage/0.2.29",
```

Comment:
Use `"url": "https://www.nuget.org/api/v2/package/git-istage/0.2.29#/dl.nupkg"` And remove that Move-Itemon L20.

Reply:
Thanks for your comments.

I tried that first, but then scoop automatically runs 7zip and extracts the package. But the compressed package is needed when `dotnet tool install ...` is run later on. Actually I was quite surprised that scoop "knows" that .nupkg is a zip file, or maybe it just tries?

Reply:
Sorry my bad I forgot. Does dotnet cli check for extension?? If not just rename it to smething that is not any of these:

https://github.com/lukesampson/scoop/blob/915eef0d0210c976840e30685f1e0dace674914c/lib/decompress.ps1#L17